libbeat/docs/upgrading.asciidoc

```diff
@@ -141,9 +141,29 @@ template to 7.0.
 migration.6_to_7.enabled: true
 ----
 
-The field aliases enable you to use 6.x dashboards and visualizations
-with indices created by {beats} 7.0 or later. The aliases do *not* work
-with saved searches or with API calls that manipulate documents directly.
+The field aliases let you use 6.x dashboards and visualizations with indices
+created by {beats} 7.0 or later. The aliases do *not* work with saved searches
+or with API calls that manipulate documents directly. 
+
+Some fields also have type changes in 7.0 that affect the behavior of older
+dashboards and visualizations. To clarify:
+
+* *Some fields have type changes.* Your {kib} visualizations and aggregations 
+will not work on these fields until the conflicts are resolved.
+
+* *Some fields have name changes, but no type changes.* The field aliases
+created by the compatibility layer ensure that visualizations and aggregations
+on the old field names work on old and new data. 7.0 dashboards only
+work on the new field names, and therefore only work with the new data.
+
+* *Some fields have both name and type changes.* Field aliases are created for
+these fields, but your {kib} visualizations and aggregations will not work on
+these fields until the conflicts are resolved. Some of your  {es} API queries
+may continue to work, if the old and new types are compatible.
+
+// TODO: Add a link to the breaking changes after we've added the list of type
+// changes. 
+// TODO: Add link to docs about resolving conflicts when they are available.
```

Comment:
I wonder if people from the field or support (knowledge base) have something about conflicts already.

This could really help kick-start a documentation page on this. I'm sure they'd be happy to help clarify this in the official docs, as it would potentially reduce the support issues around conflicts.

Reply:
I will follow up and get the ball moving on this.